lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -86,37 +92,55 @@ def configure(conf)
         end
       end
 
-      unless @use_metadata_service
-        unless @project_id && @zone && @vm_id
-          raise Fluent::ConfigError,
-              ('Please specify "project_id", "zone" and "vm_id" if you set "use_metadata_service" to false')
+      if @use_metadata_service
+        if @project_id.nil?
+          begin
+            @project_id = fetch_metadata('project/project-id')
+          rescue
```

Comment:
This code structure will hide errors while fetching metadata.
Is there a way to easy debug that if we don't propagate the errors?